### ISSUE TITLE
Explain relu and softmax in episode02 (Classification by a Neural Network using Keras)

### ISSUE THREAD

**Issue by colinsauze** [carpentries-incubator/deep-learning-intro#101]
A callout explaining relu and softmax might be helpful in episode02 in the create a network stage. These two terms are suddenly dropped in without any prior explanation. I know its not essential they are understood at this point but some learners are curious, hence the suggestion of a callout.

**Comment by bpmweel** [comment]
Also see #119

**Comment by colinsauze** [comment]
I think this was supposed to be fixed in PR #150 but I'm not seeing it either in the commit history or the final page and I think it might have been missed.
